python/dgl/nn/pytorch/gt/lap_enc.py

```diff
@@ -0,0 +1,160 @@
+"""Laplacian Positional Encoder"""
+
+import torch as th
+import torch.nn as nn
+
+
+class LapPosEncoder(nn.Module):
+    r"""Laplacian Positional Encoder (LPE), as introduced in
+    `GraphGPS: General Powerful Scalable Graph Transformers
+    <https://arxiv.org/abs/2205.12454>`__
+
+    This module is a learned laplacian positional encoding module using
+    Transformer or DeepSet.
+
+    Parameters
+    ----------
+    model_type : str
+        Encoder model type for LPE, can only be "Transformer" or "DeepSet".
+    num_layer : int
+        Number of layers in Transformer/DeepSet Encoder.
+    k : int
+        Number of smallest non-trivial eigenvectors.
+    lpe_dim : int
+        Output size of final laplacian encoding.
+    n_head : int, optional
+        Number of heads in Transformer Encoder.
+        Default : 1.
+    batch_norm : bool, optional
+        If True, apply batch normalization on raw LaplacianPE.
+        Default : False.
+    num_post_layer : int, optional
+        If num_post_layer > 0, apply an MLP of ``num_post_layer`` layers after
+        pooling.
+        Default : 0.
+
+    Example
+    -------
+    >>> import dgl
+    >>> from dgl import LaplacianPE
+    >>> from dgl.nn import LapPosEncoder
```

Comment:
Why did you want to break the consistency of using `Laplacian` for `PE` and `PosEncoder`?

Reply:
My consideration here is to keep consistency with `SpatialEncoder`, `DegreeEncoder`, etc. However, `LaplacianPosEncoder` seems too long. Any suggestion?

Reply:
Alternatively, you can rename `LaplacianPE` to `LapPE`. To maintain backward compatibility, consider having an alias for now and adding a deprecation warning.